frontend/integration-tests/preload.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-undef */
+
+import { addAlias } from 'module-alias';
+
+// The `lodash-es` package ships code that uses ECMAScript modules. Since `ts-node`
+// excludes everything under `node_modules` directory from compilation, attempting
+// to import `lodash-es` code yields syntax errors.
```

Comment:
Both unit and integration tests use `lodash` instead of `lodash-es` package.

```
$ ack -r -l --ignore-dir=".cache-loader,node_modules" 'from '"'"'lodash(?!-es)' .
__tests__/components/operator-lifecycle-manager/catalog-source.spec.tsx
__tests__/components/operator-lifecycle-manager/install-plan.spec.tsx
__tests__/components/operator-lifecycle-manager/subscription.spec.tsx
__tests__/extend/kubevirt/module/k8s/vms.spec.js
integration-tests/tests/deploy-image.scenario.ts
integration-tests/tests/environment.scenario.ts
integration-tests/tests/filter.scenario.ts
integration-tests/tests/modal-annotations.scenario.ts
integration-tests/tests/olm/update-channel-approval.scenario.ts
integration-tests/tests/olm/etcd.scenario.ts
integration-tests/tests/source-to-image.scenario.ts
integration-tests/tests/crud.scenario.ts
integration-tests/views/crud.view.ts
integration-tests/protractor.conf.ts
```

Application code in `public` directory uses `lodash-es` in order to get better webpack optimization (tree shaking of ES modules).

Both unit and integration tests are executed within a Node.js (v8 LTS) env. `ts-node` avoids compiling files in `node_modules` directory ([link](https://github.com/TypeStrong/ts-node#how-it-works)). So unless the test runner explicitly supports transpilation of source files, importing code directly from `node_modules` may result in syntax errors.

Jest is configured with [`ts-jest`](https://github.com/kulshekhar/ts-jest) transformer applied to `.(ts|tsx|js|jsx)` when loading *any* kind of module, i.e. including stuff in `node_modules`. Your Jest config also includes:

```
"transformIgnorePatterns": [
  "<rootDir>/node_modules/(?!lodash-es/.*)"
],
```

which means that Jest won't transform anything under `node_modules/lodash-es`.

On the other hand, Protractor doesn't seem to have any file transformation capability - we simply rely on `ts-node` to augment/hack the default Node.js module loading impl. by running `tsc` on files with `.tsx?` and `.jsx?` extensions. _Again, `ts-node` does not process stuff in `node_modules`._

So, this problem occurs only in integration tests when trying to load (import) a module from `node_modules` directory which doesn't conform to supported Node.js (v8 LTS) syntax, such as `lodash-es`.

@spadgett Try adding following line into `integration-tests/tests/crud.scenario.ts`:

```tsx
import { referenceForModel } from '../../public/module/k8s/k8s';
```

and running `yarn test-suite --suite crud` - Protractor will fail:

```
E/launcher - Error: /home/vszocs/go/src/github.com/openshift/console/frontend/node_modules/lodash-es/lodash.js:10
export { default as add } from './add.js';
^^^^^^

SyntaxError: Unexpected token export
    at new Script (vm.js:79:7)
    ....
```

This PR fixes the problem by adding `lodash-es` :arrow_right: `lodash` alias specifically for integration tests via `integration-tests/preload.ts`.

(It also fixes another related problem where expressions like `SERVER_FLAGS.whatever` throw because the global `SERVER_FLAGS` is not defined.)

Reply:
> Is there a reason we don't see errors for all of our integration tests?

None of the integration tests so far attempted to import code that eventually loaded `lodash-es` module.

Reply:
Also, this PR adds an explicit dev-dependency on `lodash`. Up to now, `lodash` was pulled into `node_modules` through some other dependencies.